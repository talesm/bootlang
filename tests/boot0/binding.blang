let a = 5;
writeln(a.toString());
let mutable s = 'Hi';
writeln(s);
s = 'Hello';
writeln(s);
let b = number.parseOrPanic(readln());
let c = number.parseOrPanic(readln());
writeln(b.add(c).toString());