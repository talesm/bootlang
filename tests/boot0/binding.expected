5
Hi
Hello
7
